src/containerapp/azext_containerapp/tests/latest/test_containerapp_maintenance_config.py

```diff
@@ -58,6 +58,13 @@ def test_containerapp_maintenanceconfig_crudoperations_e2e(self, resource_group)
             JMESPathCheck('properties.scheduledEntries[0].weekDay', updatedWeekday),
         ])
 
+        # update nothing to confirm all properties remain
```

Comment:
Please rerun and update recording files, ensure the test pass without --live in the playback mode.
otherwise the PR's CI will fail too.

Reply:
updated with live test run